Podcast/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -51,7 +51,7 @@
     {
       "size" : "1024x1024",
```

Comment:
if you are changing the app icon make sure its changed in the "About Us" in Settings and also on Facebook. You need to log into Facebook via app dev's account and change it in the developer site.